agent/taskresource/volume.go

```diff
@@ -24,7 +24,6 @@ type VolumeResource struct {
 	Name                string
 	Mountpoint          string
 	Driver              string
-	DriverOptions       map[string]string
```

Comment:
My understanding was we use `VolumeResource` obj to call createVolume (equivalent to CreateResource) with its attributes. In that case, `DriverOptions` would still remain here?

Reply:
There's a separate createVolumeOptions obj: https://tinyurl.com/ya4uoo4n
This VolumeResource struct is an internal representation of volume. We may need this field back later, but we might need a PR against go-dockerclient to surface this info.